docs/topics/legal-developers-terms.md

```diff
@@ -85,7 +85,7 @@ Please read these Terms carefully. These Terms govern your access to and use of
             <li><strong>Termination</strong>. Either party may terminate these Terms upon 30 days notice for any reason. In addition, Overwolf may terminate these Terms without notice if it has justifiable reason to believe that Developer is in material breach of any provision hereof. Developer understands that Overwolf requires the ability to terminate without notice in the event of Developer’s material breach in order to protect Overwolf from liability.</li>
-            <li><strong>Effect of Termination</strong>. Upon termination of these Terms for any reason Developer shall cease the use of the rights licensed hereunder including, for the avoidance of doubt, (a) cease all use of the Platform and Platform API, (b) cease all distribution of the Application and (c) ensure that no End User may access any Application. Upon termination of these Terms for any reason, Developer shall pay to Overwolf all amounts due hereunder as of the date of termination. Termination of these Terms shall not affect the obligations of Developer to pay Overwolf all amounts owing or to become owing to Overwolf hereunder on or before the date of such termination, as well as interest thereon to the extent any such amounts are paid after the date they became or will become due pursuant to these Terms. Any revenue received after the termination of these Terms in respect of or connected to the Application shall continue to be shared among the parties according to the provisions hereof.</li>
+            <li><strong>Effect of Termination</strong>. Upon termination of these Terms for any reason Developer shall cease the use of the rights licensed hereunder including, for the avoidance of doubt, (a) cease all use of the Platform and Platform API, (b) cease all distribution of the Application and (c) ensure that no End User may access any Application. Notwithstanding the foregoing, for all User Content known as “add-ons”, “maps”, “mods”, Overwolf may continue to retain (but not distribute or display) such Content on its own servers to the extent necessary to ensure the continued functionality of Applications downloaded by end users. Upon termination of these Terms for any reason, Developer shall pay to Overwolf all amounts due hereunder as of the date of termination. Termination of these Terms shall not affect the obligations of Developer to pay Overwolf all amounts owing or to become owing to Overwolf hereunder on or before the date of such termination, as well as interest thereon to the extent any such amounts are paid after the date they became or will become due pursuant to these Terms. Any revenue received after the termination of these Terms in respect of or connected to the Application shall continue to be shared among the parties according to the provisions hereof.</li>
```

Comment:
Hello, I am the creator of https://curseforge-ideas.overwolf.com/ideas/CF-I-222, and I have some feedback.

Unless I'm misunderstanding something, according to these terms, mod authors are not allowed to distribute their mod via ANY means anymore if the terms are terminated.

Something like


> <li><strong>Effect of Termination</strong>. Upon termination of these Terms for any reason Developer shall cease the use of the rights licensed hereunder including, for the avoidance of doubt, (a) cease all use of the Platform and Platform API, (b) cease all distributions of the Application which still use the Platform and Platform API (c) ensure that no End User may access the Platform and Platform API through Developer's Application(s).



Would allow mod authors to distribute their mods outside of overwolf after termination.

Reply:
Secondly, it doesn't seem possible in these terms for a mod author to remove one mod on overworlf without being forced to terminate the terms entirely. This has a side effect that the mod author must remove **_all_** of their other mods, because application is defined as

> <li>Application means <strong>any application</strong> created by Developer through use of the Platform API and/or any software development kit provided to you by Overwolf.</li>